src/CMakeLists.txt

```diff
@@ -1,12 +1,16 @@
 
 if(CMAKE_Fortran_COMPILER_ID MATCHES "^(Intel)$")
   set(CMAKE_Fortran_FLAGS
-      "-g -traceback -free -xHOST -convert big_endian -assume byterecl")
+      "-g -traceback -free -xHOST -convert big_endian -assume byterecl ${CMAKE_Fortran_FLAGS}")
```

Comment:
should we remove the `-xHOST`.
I remember @climbfuji had a pretty good and compelling argument.

Reply:
Yes, I think so. It causes problems on machines like Jet which have multiple CPU types, and I doubt gfsio has any sort of performance reason to need it.

Reply:
> should we remove the `-xHOST`.
> I remember @climbfuji had a pretty good and compelling argument.

Yes, please. We should do that for ALL NCEPLIBS (this may change the results for the ufs-weather-model regression tests, but as long as we make this a single change to the model that is fine). Alternatively, add specific flags like
```
-axSSE4.2,AVX,CORE-AVX2
```
(note: we use `-axSSE4.2,AVX,CORE-AVX2,CORE-AVX512` for the ufs-weather-model on jet)

Reply:
I don't think gfsio even does any math. It's just an I/O layer.